### PR TITLE
Update keploy.sh

### PR DESCRIPTION
**Pull request by Harshjosh361**
fix: #2327 

Added chmod commands  to ensure no permission issues.

### CODE REVIEW

**Comment by Harshjosh361** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Sonichigo** [comment]
hi @Harshjosh361 thank you for the contribution! @Sarthak160 can you please take a look?

File: keploy.sh [hunk 1]
```diff
@@ -33,10 +35,96 @@ installKeploy (){
             ;;
         esac
     done
+
+    # Function to check for macOS version compatibility
+    check_macos_version() {
```
Comment (Sarthak160) [inline keploy.sh:40]: Hey @Harshjosh361 We dont require to check mac version for the compatibilty. Also we also have a binary for mac so we don't require to set the alias expliciltly.

**Comment by shivamsouravjha** [comment]
closing this as PR seems to be inactive. Feel free to open with request changes

**Pull request closed without merging**